### PR TITLE
Add home content

### LINKED ISSUE #42: Homepage: add below fold content sections
See design: https://github.com/sul-dlss/mirador_sul/issues/2

### PR DESCRIPTION
**Pull request by csbailey5t**
Adds below the fold content for the home page. Fixes #42

### CODE REVIEW

**Comment by versae** [comment]
I'm getting a weird space on the right. It's not shown by default, but I can easily pan to it. Size of my browser is 1280px wide.
screen shot 2017-04-12 at 9 25 16 am

File: app/assets/stylesheets/home.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .masthead-video {
   background-color: black;
-  position: fixed;
+  position: absolute;
   top: 50%;
```
Comment (versae) [inline app/assets/stylesheets/home.scss]: This change might be causing the appearance of a pannable area on the right when browser window width is smaller than video width.

Comment (aeschylus) [inline app/assets/stylesheets/home.scss]: Can it be fixed in a separate PR? It is likely a margin issue or a result of the parent's' width settings.

Comment (versae) [inline app/assets/stylesheets/home.scss]: Since it's being introduced in this PR, I'd say let's put it back to `fixed`, deploy, and then create an issue to properly tackle the the problem.
